cyhy_report/customer/report.mustache

```diff
@@ -571,15 +573,15 @@ As you review the report, you may have additional questions. Check out the answe
 
       \vspace*{1mm}  % blank line needed on previous line for vertical spacing
       {\RCSourceSansProSemiBoldSixteen <<#calc.address_scan_start_date_tex>>\usvardate\formatdate<<&calc.address_scan_start_date_tex>> \textemdash{} \usvardate\formatdate<<&calc.address_scan_end_date_tex>><</calc.address_scan_start_date_tex>><<^calc.address_scan_start_date_tex>>No addresses scanned yet<</calc.address_scan_start_date_tex>>}\\
```

Comment:
@jeffkause I just noticed that we probably want this changed as well (for verbiage consistency), to handle the rare case where a report is generated but no scans have completed yet:
```suggestion
      {\RCSourceSansProSemiBoldSixteen <<#calc.address_scan_start_date_tex>>\usvardate\formatdate<<&calc.address_scan_start_date_tex>> \textemdash{} \usvardate\formatdate<<&calc.address_scan_end_date_tex>><</calc.address_scan_start_date_tex>><<^calc.address_scan_start_date_tex>>No assets scanned yet<</calc.address_scan_start_date_tex>>}\\
```